Airport.js

```diff
@@ -56,24 +52,14 @@ class Airport {
     }
 
     getMilitaryPlanesWithType(militaryPlaneType) {
-        let militaryPlanesWithType = [];
         let militaryPlanes = this.getMilitaryPlanes();
-        for (let i = 0; i < militaryPlanes.length; i++) {
-            if (militaryPlanes[i].getMilitaryType() === militaryPlaneType) {
-                militaryPlanesWithType.push(militaryPlanes[i]);
-            }
-        }
+        let militaryPlanesWithType = militaryPlanes.filter(plane => plane.getMilitaryType() === militaryPlaneType);
         return militaryPlanesWithType;
     }
 
     getExperimentalPlanesWithClassificationLevel(classificationLevel) {
-        let experimentalPlanesWithClassificationLevel = [];
         let experimentalPlanes = this.getExperimentalPlanes();
-        for (let i = 0; i < experimentalPlanes.length; i++) {
-            if (experimentalPlanes[i].classificationLevel === classificationLevel) {
-                experimentalPlanesWithClassificationLevel.push(experimentalPlanes[i]);
-            }
-        }
+        let experimentalPlanesWithClassificationLevel = experimentalPlanes.filter(plane => plane.classificationLevel === classificationLevel);
```

Comment:
You don't need `militaryPlanesWithType` variable here.

Reply:
fixed